src/types/UTF8OutPipeReader.cpp

```diff
@@ -9,6 +9,8 @@
 UTF8OutPipeReader::UTF8OutPipeReader(HANDLE outPipe) :
     _outPipe{ outPipe }
 {
+    _buffer.fill(0);
+    _utf8Partials.fill(0);

```

Comment:
Consider to use pairs of empty braces for value initialization in the header.

Reply:
Yeah, that sounds better. I'll get that moved over for whatever instances of this I can find.